src/apis/config.js

```diff
@@ -0,0 +1,3 @@
+export const FLICKR_PUBLIC = 'fbdc58683e712c507e300e24af6ee459&format=json&nojsoncallback=1';
```

Comment:
We should move these into a file labeled `constants` or `types`.

Reply:
I'll have this changed to use `constants`.